internal/validation/validation_test.go

```diff
@@ -30,6 +30,11 @@ func TestValidate(t *testing.T) {
 				},
 				NotEmptyField:   "certainly not.",
 				MatchRegexField: "abc",
+				OneOfField:      "one",
```

Comment:
clever idea with these fields and the validation! I think the fields should be optional though to keep the focus of the test struct on the field that is relevant